examples/create-react-app/README.md

```diff
@@ -1,28 +1,28 @@
-![React Logo](../.github/images/react.svg)
+![React Logo](../packages/frameworks/logos/react.svg)
 
-# Create-React-App Example
+# React Example
 
-This directory is a brief example of a [React](https://reactjs.org/) app that can be deployed with ZEIT Now and zero configuration.
+This directory is a brief example of a [React](https://reactjs.org/) app with [Serverless Functions](https://zeit.co/docs/v2/serverless-functions/introduction) that can be deployed with ZEIT Now and zero configuration.
 
 ## Deploy Your Own
 
-Deploy your own React project with ZEIT Now.
+Deploy your own React project, along with Serverless Functions, with ZEIT Now.
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/now-examples/tree/master/create-react-app)
+[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/now-examples/tree/master/create-react-app-functions)
 
-_Live Example: https://create-react-app.now-examples.now.sh_
+_Live Example: https://create-react-app-functions.now-examples.now.sh/_
```

Comment:
Is this correct? The readme is in the `create-react-app` directory but the example is `create-react-app-functions` domain?

Reply:
Yes, `create-react-app-functions` has been renamed to `create-react-app` and the other one has been deleted, so the domain is correct

Reply:
But I've deployed this one again and adjusted the URL